samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -35,19 +35,80 @@ BUILD_ASSERT(CONFIG_BT_ISO_TX_BUF_COUNT >= TOTAL_BUF_NEEDED,
 #if defined(CONFIG_BAP_BROADCAST_16_2_1)
 
 static struct bt_bap_lc3_preset preset_active = BT_BAP_LC3_BROADCAST_PRESET_16_2_1(
-	BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT,
+	BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
```

Comment:
Suggest to move the location and context into 2 new macros so that we can reuse the values for all presets.

Reply:
when we only have 2, the gain is not huge.  Maybe future PR when/if we have more presets?

Reply:
Sure :) Simply a suggestion to reduce code duplication